app/mailers/claims/user_mailer.rb

```diff
@@ -0,0 +1,61 @@
+class Claims::UserMailer < Claims::ApplicationMailer
+  def user_membership_created_notification(user, organisation)
+    notify_email to: user.email,
+                 subject: t(".subject", service_name:),
+                 body: t(
+                   ".body",
+                   user_name: user.first_name,
+                   organisation_name: organisation.name,
+                   service_name:,
+                   heading: t(".heading"),
+                   support_email:,
+                   sign_in_url:,
+                 )
+  end
+
+  def user_membership_destroyed_notification(user, organisation)
+    notify_email to: user.email,
+                 subject: t(".subject", service_name:),
+                 body: t(
+                   ".body",
+                   user_name: user.first_name,
+                   organisation_name: organisation.name,
+                   service_name:,
+                   support_email:,
+                 )
+  end
+
+  def claim_submitted_notification(user, claim)
+    link_to_claim = claims_school_claim_url(id: claim.id, school_id: claim.school.id)
+
+    notify_email to: user.email,
+                 subject: t(".subject"),
+                 body: t(
+                   ".body",
+                   user_name: user.first_name,
+                   organisation_name: claim.school.name,
+                   reference: claim.reference,
+                   amount: claim.amount.format(symbol: true, decimal_mark: ".", no_cents: false),
+                   support_email:,
+                   link_to_claim:,
+                 )
+  end
+
+  def claim_created_support_notification(claim, user)
+    link_to_claim = claims_school_claim_url(id: claim.id, school_id: claim.school.id)
+
+    notify_email to: user.email,
+                 subject: t(".subject"),
+                 body: t(
+                   ".body",
+                   user_name: user.first_name,
+                   organisation_name: claim.school.name,
+                   reference: claim.reference,
+                   amount: claim.amount.format(symbol: true, decimal_mark: ".", no_cents: false),
+                   support_email:,
+                   link_to_claim:,
+                 )
+  end
+end
+
+class InvalidOrganisationError < StandardError; end
```

Comment:
What's this class needed for? 🤔

I'm reviewing this on my phone, so it might be easier to see on a laptop. It feels like a bit of a "WTF" though.